aten/src/ATen/native/cuda/UpSampleBilinear2d.cu

```diff
@@ -218,14 +390,15 @@ static void upsample_bilinear2d_backward_out_cuda_template(
   int input_height = input_size[2];
   int input_width = input_size[3];
 
-  Tensor grad_output = grad_output_.contiguous();
-
   if (grad_input.numel() == 0) {
     return;
   }
 
-  // A contiguous tensor is required for the kernel launch config
-  grad_input.contiguous();
+  const auto memory_format = grad_output_.suggest_memory_format();
```

Comment:
probably better to infer format from grad_input?  grad_output could have lost channels last